packages/usa-file-input/src/usa-file-input.stories.js

```diff
@@ -16,6 +16,16 @@ export default {
       control: { type: "radio" },
       options: ["none", "disabled", "aria-disabled"],
     },
+    error_message: {
+      name: "Error message",
+      control: { type: "text" },
+      defaultValue: "",
+    },
+    accepted_files_message: {
+      name: "Accepted files message",
+      control: { type: "text" },
+      defaultValue: "",
+    },
```

Comment:
**issue:** I found the error controls in these stories to be confusing. I was trying to customize the "Error message" field thinking that it would represent `data-errorMessage`. I didn't expect that data attribute to be represented by a control named "Accepted files message". Can you add some clarity to the control names here? 

If we could also rename the "error" control to something more descriptive ("Form group error styles"?), that would be helpful as well. 

Feel free to edit this, but here is an idea:

```suggestion
    error_hint_text: {
      name: "Error text - hint",
      control: { type: "text" },
      defaultValue: "",
    },
    invalid_file_text: {
      name: "Error text - Invalid file type (data-errorMessage)",
      control: { type: "text" },
      defaultValue: "",
    },
```